AEPCore/Sources/eventhub/ExtensionContainer.swift

```diff
@@ -61,7 +61,7 @@ class ExtensionContainer {
     init(_ name: String, _ type: Extension.Type, _ queue: DispatchQueue, completion: @escaping (EventHubError?) -> Void) {
         extensionQueue = queue
         containerQueue = DispatchQueue(label: "\(name).containerQueue")
-        eventOrderer = OperationOrderer<Event>()
+        eventOrderer = OperationOrderer<Event>(String(describing: type))
```

Comment:
You can use "\(name).operationOrderer"

Reply:
I tested that:
name = "com.adobe.eventhub.extension.AEPCore.Configuration"	
String(describing: type) = "Configuration"

Reply:
Yes. For the sake of consistency with the previous line, we can reuse the name variable. If we decide to change, we need to use a consistent name for extensionQueue, containerQueue, and operationOrderer, preferably with a com.adobe prefix..